dialer.go

```diff
@@ -12,6 +13,12 @@ import (
 	"github.com/streadway/amqp"
 )
 
+
+const (
+	errMsgReadyChanUnbuffered   = "ready chan is unbuffered"
+	errMsgUnreadyChanUnbuffered = "unready chan is unbuffered"
```

Comment:
if something starts with `err` I would expect an error, not a string. Could you create an error using `errors.New("")`?

Reply:
const removed